### PR TITLE
fix: Prevent overflow when rendering expression hints

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/src/components/ParameterInputHint.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
 	<n8n-text size="small" color="text-base" tag="div" v-if="hint">
-		<div v-if="!renderHTML" :class="classes">{{ hint }}</div>
+		<div v-if="!renderHTML" :class="classes"><span v-html="simplyText"></span></div>
```
Comment (mutdmour) [inline packages/editor-ui/src/components/ParameterInputHint.vue:3]: tried to tests based on work before done by Ivan but no luck..

**Review by OlegIvaniv — DISMISSED** [review]
Works as expected 👌

File: cypress/e2e/14-mapping.cy.ts [hunk 2]
```diff
@@ -205,63 +205,63 @@ describe('Data mapping', () => {
...
-
-		ndv.actions.mapToParameter('fieldName');
-
-		ndv.getters.inlineExpressionEditorInput().should('have.length', 0);
-		ndv.getters
-			.parameterInput('fieldName')
-			.find('input')
-			.should('have.value', "input[0]['hello.world']['my count']");
-	});
+	// it('maps keys to path', () => {
```
Comment (OlegIvaniv) [inline cypress/e2e/14-mapping.cy.ts:221]: Do we still need this spec? If so, it might be worth to add a comment explaining why it's commented out

Comment (mutdmour) [reply cypress/e2e/14-mapping.cy.ts:221]: mistake

**Review by OlegIvaniv — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.228.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.228.2)

**Pull request merged**